nncf/experimental/post_training/algorithms/fast_bias_correction/algo_backend.py

```diff
@@ -0,0 +1,70 @@
+"""
```

Comment:
I would suggest the following naming convention:
- Files:
  - `backend.py`
  - `onnx_backend.py`
  - ...
- Classes:
  - `FBCBackend()`
  - `ONNXFBCBackend()`